package.json

```diff
@@ -1,19 +1,26 @@
 {
   "name": "ping-javascript-sdk",
+  "version": "0.0.0",
   "private": true,
   "description": "Ping JavaScript SDK",
-  "packageManager": "pnpm@9.15.9+sha512.68046141893c66fad01c079231128e9afb89ef87e2691d69e4d40eee228988295fd4682181bae55b58418c3a253bde65a505ec7c5f9403ece5cc3cd37dcf2531",
-  "engines": {
-    "node": "^20 || ^22",
-    "pnpm": "9.15.9"
+  "homepage": "https://github.com/ForgeRock/ping-javascript-sdk#readme",
+  "bugs": {
+    "url": "https://github.com/ForgeRock/ping-javascript-sdk/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ForgeRock/ping-javascript-sdk.git"
   },
+  "author": "ForgeRock",
   "scripts": {
     "build": "nx affected --target=build",
-    "clean": "shx rm -rf ./{coverage,dist,docs,node_modules,tmp}/ ./{packages,e2e}/*/{dist,node_modules}/ && git clean -fX -e \"!.env*,nx-cloud.env\"",
+    "changeset": "changeset",
     "ci:release": "pnpm publish -r --no-git-checks && changeset tag",
     "ci:version": "changeset version && pnpm install --no-frozen-lockfile && pnpm format",
-    "changeset": "changeset",
+    "circular-dep-check": "madge --circular .",
```

Comment:
Added this for sanity, run it locally if you want to be safe and make sure we arent breaking the world.